nilearn/plotting/displays/_slicers.py

```diff
@@ -353,16 +368,44 @@ def add_contours(self, img, threshold=1e-6, filled=False, **kwargs):
             "colors", which is one color or a list of colors for
             these contours.
 
+        cbar_vmin : :obj:`float`, optional
```

Comment:
We should  keep the argument order, so that `filled` would come after `cbar_vmax`